recipe/meta.yaml

```diff
@@ -24,10 +24,13 @@ requirements:
   host:
     - python
     - pip
-    - openssl
-    - postgresql
     - wheel
-    - krb5  # [win]
+    # It's technically not required by the code, but the do excplitly link with -lcrypto.
+    # The linker is smart enough to not actually link it since no openssl symbol is directly used,
+    # hence why we also ad it to ignore_run_exports.
+    - openssl {{ openssl }}  # [not linux]
+    - libpq 12.15
+    - krb5 1.20.1  # [win]
```

Comment:
This is wrong... We should rebuild postgres (and libpq) so thta it has a dependency on krb5.

The libpq build says:
```
INFO (libpq,Library/bin/libpq.dll): Needed DSO $RPATH/gssapi64.dll found in the whitelist
```
And when we look at the [recipe](https://github.com/AnacondaRecipes/postgresql-feedstock/blob/master/recipe/meta.yaml#L79-L80), we can see that they are indeed ignored. That's bad.

Additionally, I see
```
INFO  WARNING (libpq,Library/bin/libpq.dll): $RPATH/libssl-3-x64.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.                                                                                                                 
INFO  .. is this binary repackaging?                                                                                                                                                                                                                  
INFO  WARNING (libpq,Library/bin/libpq.dll): $RPATH/libcrypto-3-x64.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.                                                                                                              
INFO  .. is this binary repackaging?           
```